### PR TITLE
Added style prop to tech-icon and update the test.

### PR DESCRIPTION
**Pull request by raeioul**

### CODE REVIEW

File: src/components/TechIcon.js [hunk 1]
```diff
@@ -23,13 +23,16 @@ TechIcon.propTypes = {
     width: PropTypes.string,
     height: PropTypes.string,
   }),
+  style: PropTypes.shape({
```
Comment (diegolanda) [inline src/components/TechIcon.js]: `PropTypes.object`

File: src/components/__tests__/TechIconList.js [hunk 2]
```diff
@@ -30,4 +30,5 @@ describe('Icon Component', () => {
 
     expect(children.length).toBe(2);
   })
+
```
Comment (diegolanda) [inline src/components/__tests__/TechIconList.js]: Not needed if no change

File: src/components/__tests__/TechIcon.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ describe('Icon Component', () => {
   });
 
   it('contains only one img child tag', () => {
-    const wrapper = mount(<TechIcon {...DEFAULT_PROPS}/>);
+    const wrapper = mount(<TechIcon name="android" style={padding} />);
```
Comment (diegolanda) [inline src/components/__tests__/TechIcon.js]: there is no actual test with this, 

You would need to create another test like:

it('should add style prop when passed')

it('should override style width and height if size and style props are present')

**Review by diegolanda — CHANGES_REQUESTED** [review]
Please review

File: src/components/TechIcon.js (hunk 1 above)
Comment (raeioul) [reply src/components/TechIcon.js]: Done.

File: src/components/__tests__/TechIconList.js (hunk 2 above)
Comment (raeioul) [reply src/components/__tests__/TechIconList.js]: The line was deleted.

File: src/components/__tests__/TechIcon.js (hunk 3 above)
Comment (raeioul) [reply src/components/__tests__/TechIcon.js]: The tests has been added.

Comment (diegolanda) [reply src/components/__tests__/TechIcon.js]: Leave this with `{...DEFAULT_PROPS}`

File: src/components/__tests__/TechIcon.js [hunk 4]
```diff
@@ -20,14 +25,36 @@ describe('Icon Component', () => {
...
   })
   
+  it('Should add style prop when passed', () => {
+    const wrapper = mount(<TechIcon name="android" style={padding} />);
+
+    const props = wrapper.find('img').props();
+
+     expect(props.src).toBe(`${DEFAULT_PROPS.name}.png`);
+     expect(props.alt).toBe(`${IMAGE_PATH}${DEFAULT_PROPS.name}.png`);
+     expect(props.style.padding).toBe(`${padding.padding}`);
```
Comment (diegolanda) [inline src/components/__tests__/TechIcon.js]: `expect(props.style).toBe(padding)`

File: src/components/__tests__/TechIcon.js [hunk 5]
```diff
@@ -20,14 +25,36 @@ describe('Icon Component', () => {
...
+  })
+
+  it('should override style width and height if size and style props are present', () => {
+    const wrapper = mount(<TechIcon name="android" style={padding} size={size}/>);
+
+    const props = wrapper.find('img').props();
+
+     expect(props.src).toBe(`${DEFAULT_PROPS.name}.png`);
+     expect(props.alt).toBe(`${IMAGE_PATH}${DEFAULT_PROPS.name}.png`);
+     expect(props.style.padding).toBe(`${padding.padding}`);
```
Comment (diegolanda) [inline src/components/__tests__/TechIcon.js]: Same as above

**Review by diegolanda — CHANGES_REQUESTED** [review]

**Review by diegolanda — APPROVED** [review]

**Pull request merged**